usaspending_api/submissions/management/commands/load_submission.py

```diff
@@ -289,6 +289,15 @@ def handle(self, *args, **options):
             award = load_data_into_model(Award(), row, field_map=award_field_map, value_map=award_value_map, as_dict=True)
             award, created = Award.objects.get_or_create(**award)
 
+            fad_value_map = {
+                "award": award,
+                "submission": submission_attributes
+
+            }
+
+            financial_assistance_data = load_data_into_model(FinancialAssistanceAward(), row, value_map=fad_value_map, as_dict=True)
+            fad, created = FinancialAssistanceAward.objects.get_or_create(**financial_assistance_data)
```

Comment:
We can add award ID and mod # as unique together, which should prevent any accidental duplication. Additionally, I'll update this to check for an existing pair manually beforehand.

Shouldn't all data be new data if the submission ID hasn't been in the data store before? Or is it possible one line of D2 is updated by a later line in the same D2 from Broker?